index.html

```diff
@@ -16,49 +16,61 @@
     </nav>
     
 
-    
     <!-- Main Content Area -->
     <section class="section">
         <div class="container">
             <div class="columns">
                 <main id="artistResults" class="column is-two-thirds has-background-grey">
                     <!-- Search Parameters -->
                     <form class="notification has-background-light">
+                        <div class="buttons">
+                            <div class="control">
+                                <label class="radio" value="Country">
+                                  <input id="countryBtn" type="radio" name="answer">
+                                  Country
+                                </label>
+                                <label class="radio" value="Classical">
+                                  <input id="classicalBtn" type="radio" name="answer">
+                                  Classical
+                                </label>
+                                <label class="radio" value="Electronic">
+                                    <input id="electronicBtn" type="radio" name="answer">
+                                    Electronic
+                                </label>
+                                <label class="radio" value="Hip Hop / Rap">
+                                    <input id="hiphopBtn" type="radio" name="answer">
+                                    Hip Hop / Rap
+                                </label>
+                                <label class="radio" value="Jazz">
+                                    <input id="jazzBtn" type="radio" name="answer">
+                                    Jazz
+                                </label>
+                                <label class="radio" value="Pop">
+                                    <input id="popBtn" type="radio" name="answer">
+                                    Pop
+                                </label>
+                                <label class="radio" value="Rock">
+                                    <input id="rockBtn" type="radio" name="answer">
+                                    Rock
+                                </label>
+                            </div>
+                        </div>
```

Comment:
love the radio buttons, will make connecting our API for the ticketmaster more usable